src/database/migrations/0008-create-proposal_field_values.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE proposal_field_values (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  proposal_version_id INTEGER NOT NULL,
+  application_form_field_id INTEGER NOT NULL,
+  value VARCHAR NOT NULL,
+  sequence INTEGER NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  CONSTRAINT fk_proposal_version
+    FOREIGN KEY(proposal_version_id)
+      REFERENCES proposal_versions(id),
+  CONSTRAINT fk_application_form_field
+    FOREIGN KEY(application_form_field_id)
+      REFERENCES application_form_fields(id)
+);
```

Comment:
We eventually will want an index of some kind in here. Maybe a unique constraint/index across (proposal_version_id, application_form_field, sequence)? Not that this is particularly restrictive or meaningful but it would be some index instead of no index. You could still have the same form field id repeated using sequence.

Reply:
This is a good call; I'll add the unique.